packages/shared/src/components/notifications/InAppNotification.tsx

```diff
@@ -79,15 +81,21 @@ export function InAppNotificationElement(): ReactElement {
     };
   }, [payload]);
 
-  const onNotificationClick = (id: string, type: NotificationType) => {
+  const onNotificationClick = (
+    _: React.MouseEvent<HTMLAnchorElement>,
+    id: string,
+    type: NotificationType,
+  ) => {
     trackEvent({
       event_name: AnalyticsEvent.ClickNotification,
       target_id: id,
       extra: JSON.stringify({ origin: Origin.RealTime, type }),
     });
   };
 
-  if (!payload?.notification) {
+  const isNotifTypeSubscribe =
```

Comment:
But this would already not be rendered right? So why do we bother handling the click specificly?

Reply:
Just to catch any edge case and never show any form of subscribe to notification.